PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -45,57 +47,73 @@ public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContain
         String suffix = metaData.getSuffix();
 
         inspectContainer.addValue(getWithIcon("Primary group", Icon.called("user-friends").of(Family.SOLID)), user.getPrimaryGroup());
-
-
         inspectContainer.addValue(getWithIcon("Prefix", Icon.called("file-signature").of(Family.SOLID).of(Color.GREEN)), prefix != null ? prefix : "None");
         inspectContainer.addValue(getWithIcon("Suffix", Icon.called("file-signature").of(Family.SOLID).of(Color.BLUE)),suffix != null ? suffix : "None");
 
-        TableContainer metaTable = new TableContainer(
-                getWithIcon("Meta", Icon.called("info-circle").of(Family.SOLID)),
-                getWithIcon("Value", Icon.called("file-alt").of(Family.SOLID))
-        );
-        metaData.getMeta().forEach((key, value) -> metaTable.addRow(key, value));
-        inspectContainer.addTable("Meta", metaTable);
-
-        List<String> groups = user.getPermissions().stream().filter(Node::isGroupNode).map(Node::getGroupName).collect(Collectors.toList());
+        if (metaData.getMeta().size() > 0) {
```

Comment:
`Collection#isEmpty` would be more efficient than `size > 0`, since `isEmpty` has to only check a single entry.